### ISSUE TITLE
File Input - WCGA Rule 1.4.13 Content on Hover or Focus

### ISSUE THREAD

**Issue by CillianHPE** [grommet/grommet#7562]
Issue:
Keyboard users can’t access the tooltip that shows the full file name when it gets cut off, which fails WCAG 1.4.13.

Fix:
Make the tooltip accessible with keyboard focus, or show the full file name in another accessible way (like in a focusable element or aria-label).

![Image](https://github.com/user-attachments/assets/d02b4a62-dd57-4d39-a04d-25199aeaff7a)

**Comment by CillianHPE** [comment]
@jcfilben Looking back on this issue, the screen reader does read the file name. Can you confirm if this is a pass or fail?

**Comment by jcfilben** [comment]
I think that it is still an issue because a keyboard only user who doesn't use a screenreader doesn't have a way to access the tooltip

**Comment by britt6612** [comment]
Look into using grommet tooltip and not the out of the box browser then this may pass with this fix 
https://github.com/grommet/grommet/pull/7568

**Comment by britt6612** [comment]
closing https://github.com/grommet/hpe-design-system/pull/5191# was merged
